docs(misc): Using Tailwind CSS with React Guide (#6183)
nrwl/nx: docs/react/guides/installing-tailwindcss-in-web-react-nextjs-apps.md

> # Installing TailwindCss in Web/React/Next.js Apps

Reviewer: Should name the guide `Using Tailwind CSS in React`. We can mention that it is for both Next.js and React in the details. Not need to mention Web.

> This guide will serve as a quickstart to installing TailwindCss in your Web/React/Next.js app.

Reviewer: Should be `Tailwind CSS` not `TailwindCss`. Maybe also link out to `https://tailwindcss.com/`?

> This guide will serve as a quickstart to installing TailwindCss in your Web/React/Next.js app.

Reviewer 2: ```suggestion
This guide serves as a quickstart to installing TailwindCss in your Web/React/Next.js app.
```

For more in-depth look on this topic, be sure to check out our blog post on [Setting up Next.js to use Tailwind with Nx](https://blog.nrwl.io/setup-next-js-to-use-tailwind-with-nx-849b7e21d8d0).

## Step 1: Install Tailwind Dependencies

`npm install tailwindcss@latest postcss@latest autoprefixer@latest`

> This will install the requisite tailwind dependencies.

Reviewer 2: ```suggestion
This installs the requisite tailwind dependencies.
```

## Step 2: Intialize Tailwind

> In this step, we'll create a `postcss.config.js` and a `tailwind.config.js` file specific to the application we'd like to introduce tailwind to.

Reviewer 2: ```suggestion
In this step, create a `postcss.config.js` and a `tailwind.config.js` file specific to the application to introduce Tailwind to.
```

> The simplest way to do this uses the Tailwind CLI, and can be done via:

Reviewer 2: Avoid usage of words like `via`, `etc`, `e.g.` as they aren't common across languages
```suggestion
The simplest way to do this uses the Tailwind CLI, and can be done with:
```

```bash
cd apps/{your app here}
npx tailwindcss init -p
```

> This will create the needed files for you with a general boilerplate implementation.

Reviewer 2: Avoid usage of `will`
```suggestion
This creates the needed files for you with a general boilerplate implementation.
```
